.circleci/config.yml

```diff
@@ -0,0 +1,202 @@
+version: 2.0
+
+jobs:
+  cogutil:
+    docker:
+      - image: opencog/opencog-deps
+        user: root
+        environment:
+          CCACHE_DIR: /ws/ccache
+    working_directory: /ws/cogutil
+    steps:
+      - checkout
+      - run:
+          name: Start restoring ccache
+          command: date +%d-%m-%Y > /tmp/date
+      - restore_cache:
+          key:  cogutil-ccache-{{ checksum "/tmp/date" }}
```

Comment:
If you add `key: cogutil-ccache-` as a second entry, then this will fall-back to the previous day's cache if there isn't cache for today.

There is also a double space between `key:` and `cogutil`. I'd hope circleci/yaml is smart enough to remove this, but might be safer to only use one space.